api/services/protocols/ldap.js

```diff
@@ -0,0 +1,73 @@
+var _ = require('lodash');
+var adminGroup = new RegExp(process.env.ADMIN_GROUP_REG || null);
+var commonName = /^cn=([^,]+),.*/
+
+var ldapToUser = function (ldapUser, next) {
+    var data = {
+        active: true
+    }
+    data.username = ldapUser.uid;
+    data.firstName = ldapUser.givenName;
+    data.lastName = ldapUser.sn;
+    data.email = ldapUser.mail;
+
+    sails.models.user.create(data)
```

Comment:
Where is sails defined?

Reply:
It's a magical super global that I have no clue how it gets there, but it's _everywhere_.